src/api/python/setup.py.in

```diff
@@ -0,0 +1,21 @@
+from distutils.core import setup
+from distutils.command.clean import clean
+from distutils.command.install import install
+
+class Pycvc4Install(install):
```

Comment:
```suggestion
class PyCVC4Install(install):
```

Reply:
I messed up the diff with my previous commit so I made the change here instead: ba6fa45bf7478f40768780cc7d4d5f829d3ac4ca